pyjetty/alice_analysis/analysis/user/substructure/run_analysis.py

```diff
@@ -9,25 +9,25 @@
     The grooming part is optional, and should be labeled e.g. zcut01_B0 — from CommonUtils::grooming_label({'sd':[zcut, beta]})
     For example: h_subjet_z_JetPt_R0.4_0.1
     For example: h_subjet_z_JetPt_R0.4_0.1_zcut01_B0
-    
+
```

Comment:
A lot of whitespace changes here...is this some autoformat thing? Would be good if we can avoid spamming PRs with these ;)

Reply:
Okay, will try to keep these separate in the future.